lib/infobox/main.ts

```diff
@@ -26,9 +36,12 @@ export const Main = function (sidebar, linkScale) {
 
     el.scrollIntoView(false);
     el.classList.add("infobox");
+    // @ts-ignore
```

Comment:
If `destroy()` should be kept with the element, you could use an extended element with the `destroy()` function defined.